jboss/container/wildfly/launch/elytron/added/launch/elytron.sh

```diff
@@ -241,9 +240,10 @@ EOF
 
 configure_https() {
 
-  if [ "${CONFIGURE_ELYTRON_SSL}" != "true" ]; then
-    log_info "Using PicketBox SSL configuration."
-    return
+  if [ -n "${CONFIGURE_ELYTRON_SSL}" ]; then
```

Comment:
Should we allow at least to use CONFIGURE_ELYTRON_SSL=true?

That would avoid forcing the customers who are already using elytron to configure the SSL to change their environments, making the transition leaner.

Reply:
We could warn in this case that this env variable is no more needed, I agree.

Reply:
ok, changes look good then, so giving a symbolic approval expecting we log a warn if CONFIGURE_ELYTRON_SSL=true instead of fail.

Reply:
@yersan I added the warn case.